### ISSUE TITLE
New Rule: `require-fallback-fonts`

### ISSUE THREAD

**Issue by Tanujkanti4441** [eslint/css#153]
### Rule details

Rule will enforce the use of fallback font to maintain consistency and readability across different devices and browsers.

### What type of rule is this?

Warns about a potential problem

### Example code

```css
body {
  font-family: "Open Sans";
}
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

**Comment by nzakas** [comment]
@Tanujkanti4441 what do you think of my suggestion?

Maybe the rule is called `font-family-fallbacks` and requires not only fallbacks are present but that a generic fallback is last?

**Comment by Tanujkanti4441** [comment]
Let me summarize this. Rule will be called `font-family-fallbacks` and it will report if `font-family` or `font` (if defines font) property doesn't have fallback fonts and a generic font last. It will have variable support on the same page. Is there anything I'm missing, or we can accept it?

**Comment by nzakas** [comment]
Can you define what "variable support on the same page"means?

**Comment by Tanujkanti4441** [comment]
> Can you define what "variable support on the same page"means?

Variable reference in the same stylesheet I mean.

**Comment by nzakas** [comment]
Sorry, what I mean is, what is your strategy for checking variables?

**Comment by nzakas** [comment]
Sounds good to me. Just wanted to make sure we had this behavior documented. :+1:
